### ISSUE TITLE
Differentiate cities by size

### ISSUE THREAD

**Issue by v4nz666** [v4nz666/7drl2017#58]
There are four types of cities, would be nice to differentiate them by icon.

If not this, then add information about their size to the map view, when hovering over a city with the cursor.

**Comment by Break-the-silence** [comment]
hell yea..do you mean, larger cities would have like two or three tiles of icon..combine a few tiles for say a large ass city..or 2 for medium..or is there a way to combine tiles so I could make a bigger city tile?

And of course..have different music for each of the city types

**Comment by v4nz666** [comment]
No, would have to be a single icon for each city size... maybe something like a flag, teepee, house, castle sort of deal...
